app/Http/Controllers/UserController.php

```diff
@@ -128,10 +130,9 @@ public function getUsers(User $user, Request $req) {
         return response(json_encode($toReturn), 200);
     }
 
-    public function getUser(User $user) {
-        $id = Input::get('id');
-        $user = $user->with('antenna')->findOrFail($id);
+    public function getUser(Request $req, User $user, RolesRepository $repo) {
 
+        dd($repo->getRoles($req, $user));
         $toReturn['success'] = 1;
```

Comment:
Agree! I did not aim for the standardized API in this PR, yet made some changes in responses that should make implementing the standardized API easier to implement.